### PR TITLE
Upgrade Forgelin dep to latest version

### PR DESCRIPTION
**Pull request by AlexCouch**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
 kotlin.incremental=true
-kotlin_version=1.1.2
+kotlin_version=1.2.10
```
Comment (Bluexin) [inline gradle.properties:2]: Forgelin is currently using kotlin 1.2.0

File: build.gradle [hunk 2]
```diff
@@ -101,7 +101,8 @@ dependencies {
     //required 'com.github.Eladkay:Mixin:-SNAPSHOT'
 // todo once mcmultipart is 1.11
 //    deobfCompile "MCMultiPart:MCMultiPart:1.2.1:universal"
-    compile "net.shadowfacts:Forgelin:1.5.1"
+//I'm also setting this to use the latest Forgelin cause the latest Forgelin has the latest kotlin
```
Comment (Bluexin) [inline build.gradle:104]: This comment is useless. We all know why Forgelin has to be updated ;p

File: ConvertAssets.py [hunk 3]
```diff
@@ -1,164 +0,0 @@
-#!/usr/bin/python
```
Comment (Bluexin) [inline ConvertAssets.py:1]: This file doesn't hurt anyone by being there, if people prefer having their python script to convert CamelCase to snake_case, then so be it.
It does not add any setup time or anything for other people.

**Review by Bluexin — CHANGES_REQUESTED** [review]
On a sidenote, the build has to be fixed ;p

**Comment by yrsegal** [comment]
Deleting ConvertAssets.py is a definite no-accept.

**Review by yrsegal — CHANGES_REQUESTED** [review]
Don't delete convertassets, don't comment extraneously

**Review by Eladkay — CHANGES_REQUESTED** [review]
Don't fix what ain't broke ay? That specifically refers to updating the gradle, which can not anything but issues, and don't delete the convert assets py.

**Comment by Bluexin** [comment]
Duh, gh, y u no cloz

**Pull request closed without merging**